experiment/measurer/measure_manager.py

```diff
@@ -130,8 +143,12 @@ def measure_all_trials(experiment: str, max_total_time: int, pool, q) -> bool:
     if not unmeasured_snapshots:
         return False
 
+    process_specific_df_containers = \
```

Comment:
Try to break this up on multiple lines without a backslash please (I think `presubmit.py format` will help here).